docs/quickstart-guide/quick-install.md

```diff
@@ -0,0 +1,58 @@
+# Spin up a Percona Everest instance
+
+As soon as you create a Kubernetes cluster on Amazon Elastic Kubernetes Service (EKS) or Google Kubernetes Engine (GKE), you can spin up a Percona Everest instance. For more information, see [Prerequisites](qs-prerequisites.md). 
+{.power-number}
+
+1. To spin up a Percona Everest instance, run the following command:
+
+    ```bash
+    curl -sfL "https://raw.githubusercontent.com/percona/percona-everest-cli/v0.2.0/install.sh" | bash
```

Comment:
Don't we use the prompt $ here? Why not?

Reply:
added